frontend/src/component/styles/shadow.scss

```diff
@@ -9,3 +9,5 @@ $shadow-floating-box: 0 4px 6px 0 rgba(32, 33, 36, 0.28);
 $shadow-header: 0 3px 6px color-transparent-black(0.16),
   0 3px 6px rgba(0, 0, 0, 0.23);
 $shadow-drawer: 0 0 6px color-transparent-black(0.2);
+$shadow-toggle: 0 1px 3px color-transparent-black(0.12),
```

Comment:
Currently reusing `Button`'s shadow. Seems like it'd work fine for `Toggle`'s shadow too, but having it as its own SCSS variable will allow it to be updated independently of `Button`'s.